<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="26863" started-at="2020-05-13T14:20:47Z" finished-at="2020-05-13T14:21:14Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="26863" started-at="2020-05-13T14:20:47Z" finished-at="2020-05-13T14:21:14Z">
      <class name="com.nms.radwinAutomation.login.AdminTest">
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method, java.lang.String, org.testng.ITestContext)[pri:0, instance:com.nms.radwinAutomation.login.AdminTest@e056f20]" name="setUp" is-config="true" duration-ms="6503" started-at="2020-05-13T14:20:48Z" finished-at="2020-05-13T14:20:54Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.nms.radwinAutomation.login.AdminTest.createCredentialProfile(java.util.Map)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[org.testng.TestRunner@3703bf3c]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="createCredentialProfile(java.util.Map)[pri:1, instance:com.nms.radwinAutomation.login.AdminTest@e056f20]" name="createCredentialProfile" duration-ms="18920" started-at="2020-05-13T14:20:54Z" data-provider="csvReader" finished-at="2020-05-13T14:21:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{name=Test Automation, no=1, community=public, department=NYPD, expectedMessage=Credential Profile created successfully}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createCredentialProfile -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.nms.radwinAutomation.login.AdminTest@e056f20]" name="tearDown" is-config="true" duration-ms="841" started-at="2020-05-13T14:21:13Z" finished-at="2020-05-13T14:21:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=createCredentialProfile status=SUCCESS method=AdminTest.createCredentialProfile(java.util.Map)[pri:1, instance:com.nms.radwinAutomation.login.AdminTest@e056f20] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="createDiscoveryProfile(java.util.Map)[pri:2, instance:com.nms.radwinAutomation.login.AdminTest@e056f20]" name="createDiscoveryProfile" duration-ms="0" started-at="2020-05-13T14:21:14Z" data-provider="csvReader" finished-at="2020-05-13T14:21:14Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.RuntimeException: File src\test\resources\dataproviders\AdminTest\createDiscoveryProfile.csv was not found.
[Ljava.lang.StackTraceElement;@5f577419]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.RuntimeException: File src\test\resources\dataproviders\AdminTest\createDiscoveryProfile.csv was not found.
[Ljava.lang.StackTraceElement;@5f577419
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:49)
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:131)
at org.testng.internal.Parameters.handleParameters(Parameters.java:706)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:924)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.lang.RuntimeException: File src\test\resources\dataproviders\AdminTest\createDiscoveryProfile.csv was not found.
[Ljava.lang.StackTraceElement;@5f577419
at com.nms.radwinAutomation.base.CsvDataProviders.csvReader(CsvDataProviders.java:42)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
... 26 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDiscoveryProfile -->
      </class> <!-- com.nms.radwinAutomation.login.AdminTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
